src/jumpstart/selectors.ts

```diff
@@ -11,3 +15,8 @@ export const showJumstartClaimError = (state: RootState) => {
 export const jumpstartSendStatusSelector = (state: RootState) => {
   return state.jumpstart.depositStatus
 }
+
+export const getJumpstartContractAddress = (networkId: NetworkId) => {
```

Comment:
this feels a little out of place in a selector file since it doesn't relate to redux. i see it is only used in the utils file, can we remove this function here and use the logic directly in the util?

Reply:
Sure, I can move this function.
I saw this example: https://github.com/valora-inc/wallet/blob/409ace74227f31ec46bb0015c3b44bad28c65daa/src/tokens/selectors.ts#L522
So I thought it was ok to leave this method here.